### PR TITLE
feature(yzj): add atari visualization option, fix mcts pree sampled bug (value_prefix/reward) and add atari env's pytest

### PR DESCRIPTION
**Pull request by jayyoung0802**

### CODE REVIEW

File: zoo/atari/config/atari_eval_config.py [hunk 1]
```diff
@@ -12,12 +8,24 @@
     point to the ckpt file of the pretrained model, and an absolute path is recommended.
     In LightZero, the path is usually something like ``exp_name/ckpt/ckpt_best.pth.tar``.
     """
-    model_path = "./ckpt/ckpt_best.pth.tar"
+    # sez
+    from atari_sampled_efficientzero_config import main_config, create_config
```
Comment (puyuan1996) [inline zoo/atari/config/atari_eval_config.py:12]: 注释写 以 sampled efficientzero的config为例哈

File: zoo/atari/tests/test_atari_lightzero_env_visualization.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            action = random.choice([0,1,2,3])
+            obs, reward, done, info = env.step(action)       
+            score+=reward
+            if done:
+                break
+        print('Score:{}'.format(score))
+        env.close()
+
+    def test_lightzero_env(self):
+        create_config.env_manager.type = 'base'  # when visualize must set as  base
```
Comment (puyuan1996) [inline zoo/atari/tests/test_atari_lightzero_env_visualization.py:25]: 注释用gpt优化一下

**Review by puyuan1996 — CHANGES_REQUESTED** [review]

File: zoo/atari/tests/test_atari_lightzero_env_visualization.py (hunk 2 above)
Comment (jayyoung0802) [reply zoo/atari/tests/test_atari_lightzero_env_visualization.py]: done!

File: zoo/atari/config/atari_eval_config.py (hunk 1 above)
Comment (jayyoung0802) [reply zoo/atari/config/atari_eval_config.py:12]: done!

**Pull request merged**